shared/search/user-search/render.native.js

```diff
@@ -0,0 +1,122 @@
+  const indexOfMatch = text.indexOf(match)
+  if (indexOfMatch > -1) {
+    return (
+      <Box style={globalStyles.flexBoxRow}>
+        <Text type={textType} style={style}>{text.substring(0, indexOfMatch)}</Text>
+        <Text type={textType} style={{...globalStyles.fontBold, ...style, ...emboldenStyle}}>{match}</Text>
+        <Text type={textType} style={style}>{text.substring(indexOfMatch + match.length)}</Text>
+      </Box>
+    )
+  }
+
+  return <Text type={textType} style={style}>{text}</Text>
+}
+
+function KeybaseResultBody ({username, searchText, isFollowing}) {
+  return <EmboldenTextMatch text={username} match={searchText} textType={'Body'} style={{color: isFollowing ? globalColors.green2 : globalColors.orange}} />
+}
+
+function ExternalResultBody ({username, searchText}) {
+  return <EmboldenTextMatch text={username} match={searchText} textType={'Body'} style={{color: globalColors.black_75}} />
+}
+
+function KeybaseExtraInfo ({username, fullName, isFollowing, searchText}) {
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'flex-end'}}>
+      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+        <Avatar size={16} style={{width: 16, marginRight: 4}} username={username} />
+        <EmboldenTextMatch text={username} match={searchText} textType={'BodySmall'} style={{color: isFollowing ? globalColors.green2 : globalColors.orange}} />
+      </Box>
+      {!!fullName && <Text type='BodyXSmall' style={{color: globalColors.black_40}}>{fullName}</Text>}
+    </Box>
+  )
+}
+
+function ExternalExtraInfo ({serviceUsername, fullNameOnService, icon, searchText}) {
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'flex-end'}}>
+      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+        <Icon type={icon} style={{width: 17, marginRight: 4}} />
+        <EmboldenTextMatch text={serviceUsername} match={searchText} textType={'BodySmall'} emboldenStyle={{color: globalColors.black_75}} />
+      </Box>
+      {!!fullNameOnService && <Text type='BodyXSmall' style={{color: globalColors.black_40}}>{fullNameOnService}</Text>}
+    </Box>
+  )
+}
+
+function Result ({result, searchText}: {result: SearchResult, searchText: string}) {
+  const iconStyle = {height: 32, width: 32}
+
+  let icon
+  let body = <Box />
+  switch (result.service) {
+    case 'keybase':
+      icon = <Avatar size={32} username={result.username} style={iconStyle} />
+      body = <KeybaseResultBody username={result.username} searchText={searchText} isFollowing={result.isFollowing} />
+      break
+    case 'external':
+      icon = <Icon type={result.icon} style={iconStyle} />
+      body = <ExternalResultBody username={result.username} searchText={searchText} />
+      break
+  }
+
+  // Align the body to the middle of the original 32 row so the extra info doesn't push the username down
+  const alignedBody = (
+    <Box style={{flex: 2}}>
```

Comment:
It's put along side the extra info to the right (e.g. twitter name or keybase full name). So you want this to use up all the space with the extra info to the right